src/App.jsx

```diff
@@ -42,7 +43,11 @@ function App(props) {
   } = props;
 
 
-  const addListener = () => {
+  const enableAnalytics = () => {
+    // Crash reportin
```

Comment:
typo, should be 
```suggestion
    // Crash reporting
```

:)